cli.py

```
@@ -0,0 +1,218 @@
+
+      lokikit parse /path/to/logs
+
+      lokikit parse /path/to/logs --job-name "api_logs"
+
+    Use 'lokikit discover' first to see what fields are available in your logs.
+    """
+    parse_command(ctx, directory, job_name, max_files, max_lines)
+
+
+if __name__ == "__main__":
+    cli()
```

Pylint (3.3.7)

[error] No value for argument 'ctx' in function call

(E1120)